### ISSUE TITLE
Creating directories error: Please import Comet before importing any keras modules

### ISSUE THREAD

**Issue by Hhhhhhao** [Ahmkel/Keras-Project-Template#7]
It seems like current code have little bug on comet import
according to the report, we have to import comet before keras modules

```
Create the trainer
importing trainers.CifarTrainer
getattr CifarTrainer
<class 'trainers.CifarTrainer.CifarTrainer'>
Please import Comet before importing any keras modules
```

**Comment by mannykoum** [comment]
Yeah I ended up resolving it by importing comet in main (hacky fix but not terribly bad)

**Comment by ffleader1** [comment]
> Yeah I ended up resolving it by importing comet in main (hacky fix but not terribly bad)

Can you please elaborate how did you do it. Importing comet in main will throw the error: "name 'Experiment' is not defined". Because importing Comet in _main_ means _simple_mnist_trainer.py_ is not imported with Comet sadly.

**Comment by ffleader1** [comment]
Yeah nvm.
Gotta import Comet twice, once in main and once in trainer. Wonder why this error is still not fixed.

**Comment by GanggangDong** [comment]
Yes, it seems that the module "comet_ml" should be imported at the beginning.
